openhands/core/logger.py

```diff
@@ -257,5 +257,5 @@ def _setup_llm_logger(name, log_level=logging.INFO):
     return logger
 
 
-llm_prompt_logger = _setup_llm_logger('prompt', current_log_level)
-llm_response_logger = _setup_llm_logger('response', current_log_level)
+llm_prompt_logger = _setup_llm_logger('prompt', logging.DEBUG)
+llm_response_logger = _setup_llm_logger('response', logging.DEBUG)
```

Comment:
Does this means we are hard-coding logger to DEBUG by default? Should we set INFO instead?

Reply:
The problem with INFO is, that that would make integration tests impossible as no log files would be output in CLI.

Reply:
Yes, will do that with next commit then.
Just have to keep that in mind with the current workflow optimization efforts.